main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"Rinnegan/common/database/postgres"
+	"fmt"
+	"time"
+)
+
+func main() {
```

Comment:
Let us keep this in a test module. As this is a depiciting server only, it wont be used directly right?